README.md

```diff
@@ -190,11 +190,487 @@ autotools-snap --files ./components/**/*.meta.ts
 
 - `skip-on-missing-key`: set this flag if you want to skip testing when `process.env.EYES_API_KEY` or `process.env.APPLITOOLS_API_KEY` variables are not set. By default, snap will fail if either of these keys are not set. Example usage: `snap --skip-on-missing-key`, or, `snap -s`.
 
+#### Snap Configuration
+
+Configuration is as simple as adding a `autotools.config.js` file in your `.autotools` folder.
+If you wish to use a configuration file named something different, or located somewhere else, you can specify a specific file using the `--config` flag, like so: 
+
+`yarn snap --config ./config/myConfig.js`
+
+The format of the `autotools.config.js` file is as follows:
+
+``` js
+module.exports = {
+    applitoolsApiKeyName: ‘WHATEVER_YOU_NAMED_YOUR_ENV_VARIABLE’,
```

Comment:
line 177 says: "This tool requires that the `process.env.EYES_API_KEY` value is set to your private API key."